packages/foreman/rubygem-googleauth/rubygem-googleauth.spec

```diff
@@ -1,25 +1,25 @@
-# Generated from googleauth-0.6.7.gem by gem2rpm -*- rpm-spec -*-
 # template: scl
 %{?scl:%scl_package rubygem-%{gem_name}}
 %{!?scl:%global pkg_name %{name}}
 
 %global gem_name googleauth
+%global gem_require_name %{gem_name}
 
 Name: %{?scl_prefix}rubygem-%{gem_name}
-Version: 0.13.1
-Release: 2%{?dist}
+Version: 1.1.3
+Release: 1%{?dist}
 Summary: Google Auth Library for Ruby
 Group: Development/Languages
 License: Apache-2.0
-URL: https://github.com/google/google-auth-library-ruby
+URL: https://github.com/googleapis/google-auth-library-ruby
 Source0: https://rubygems.org/gems/%{gem_name}-%{version}.gem
 
 # start specfile generated dependencies
 Requires: %{?scl_prefix_ruby}ruby(release)
-Requires: %{?scl_prefix_ruby}ruby >= 2.4.0
+Requires: %{?scl_prefix_ruby}ruby >= 2.5
 Requires: %{?scl_prefix_ruby}ruby(rubygems)
 Requires: %{?scl_prefix}rubygem(faraday) >= 0.17.3
-Requires: %{?scl_prefix}rubygem(faraday) < 2.0
+Requires: %{?scl_prefix}rubygem(faraday) < 3.a
```

Comment:
I don't think RPM supports this version number.

Reply:
So should I reverted back to `2.0`?